sw/device/tests/usbdev_stream_test.c

```diff
@@ -44,7 +44,7 @@
 #define TRANSFER_BYTES_VERILATOR 0x2400U
 
 // This is about the amount that we can transfer within a 1 hour 'eternal' test
-//#define TRANSFER_BYTES_LONG (0xD0U << 20)
+// #define TRANSFER_BYTES_LONG (0xD0U << 20)
```

Comment:
This is from `bazel run //quality:clang_format_fix`. Not sure how it got past CI in the first place, but fixing it seems harmless enough?

Reply:
Yes, someone had to fix it at some point hehe.